pkg/webhook/webhook.go

```diff
@@ -57,7 +57,7 @@ const (
 	fleetWebhookKeyFileName       = "tls.key"
 	fleetValidatingWebhookCfgName = "fleet-validating-webhook-configuration"
 	fleetGuardRailWebhookCfgName  = "fleet-guard-rail-webhook-configuration"
-	fleetWebhookSvcName           = "fleetwebhook"
+	// fleetWebhookSvcName           = "fleetwebhook"
```

Comment:
nit: comment

Reply:
Fixed. Actually I found this one when I self review. But I forget push the changes.